src/uint/boxed/encoding.rs

```diff
@@ -8,10 +8,10 @@ use core::fmt;
 /// Decoding errors for [`BoxedUint`].
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum DecodeError {
-    /// Input is not a valid size.
+    /// Input size is too small to fit in the given precision.
```

Comment:
Might need to update the text in the `Display` impl

Reply:
Done